cpp/ql/lib/semmle/code/cpp/controlflow/SSA.qll

```diff
@@ -15,6 +15,13 @@ class StandardSsa extends SsaHelper {
 }
 
 /**
+ * NOTE: If possible, prefer the SSA classes exposed by the new dataflow
+ * library:
+ * ```
+ * import semmle.code.cpp.dataflow.new.DataFlow
+ * // use `DataFlow::Definition`
```

Comment:
Needs an update to use the `Ssa::` prefix.

Reply:
Doh! Good catch!

Reply:
Fixed in 0e9286dd3412f0fed412df7ad8341d40058ac956